lib/common/http.ts

```diff
@@ -12,3 +12,98 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
+
+type HttpHeader = string[];
```

Comment:
Could be [string, string]?
https://www.typescriptlang.org/docs/handbook/basic-types.html#tuple